lib/biran/config.rb

```diff
@@ -46,8 +46,7 @@ def use_capistrano
 
     def files_to_generate
       {
-        vhost: {extension: '.conf'},
-        database: {extension: '.yml'}
+        vhost: {extension: '.conf'}
```

Comment:
so this will be backwards incompatible, hence, this will be `Biran 2.0`

Reply:
But it is not. Its not incompatible. We introduced an even bigger change when moving the files to generate from a list to a hash with more details. There is maybe one app that would be affected IF they update and even then its a simple matter of adding the config properly to the app_config.yml file. The gem still works with a minor tweak. And we aren't even on brian 1.0, yet. Not sure what the concern is. Even if you leave it in it doesn't cause an issue unless you didn't have the config/db_config.yml file in place. Which in that case, you weren't following the directions in the README about how to use the gem. Its simply a misconfiguration at that point.

Reply:
ok